tools/doc/tests/test_html.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var assert = require('assert');
```

Comment:
Nit (which means you can ignore this comment if you want to): The project has been favoring `const` over `var` for module imports like this.